lib/zeromq/majordomo/asynchronous_arbiter.rb

```diff
@@ -0,0 +1,49 @@
+require 'ffi-rzmq'
+
+module Zeromq
+  module Majordomo
+    class AsynchronousArbiter
+      INFINITE = -1
+
+      class << self
```

Comment:
This is the actual user interface for messaging: https://github.com/sittercity/arbiter/blob/develop/lib/eventer.rb#L6

As long as the `bus` responds to `publish`, it's fine. It doesn't have to be a class method.